health-services/plan-service/src/main/resources/db/migrate.sh

```
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true   -outOfOrder=true migrate
\ No newline at end of file
+flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true   -outOfOrder=true repair
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)